google/resource_dns_record_set.go

```diff
@@ -3,9 +3,11 @@ package google
 import (
 	"fmt"
 	"log"
+	"regexp"
+
+	dns "google.golang.org/api/dns/v1"
```

Comment:
Can you revert this change?

Reply:
I do need regular expressions in the validator. Am I missing something?

Reply:
Also, the dns import seems to be automatically included by gofmt (and since we do have a dns.change() call there, it seems legit.  Color me confused

Reply:
Oh sorry, I just meant that we don't need dns as a named import

Reply:
Done